packages/terra-tabs/src/Tabs.jsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ContentContainer from 'terra-content-container';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+import TabPane from './TabPane';
+import CollapsibleTabs from './_CollapsibleTabs';
+import CollapsedTabs from './_CollapsedTabs';
+import TabUtils from './TabUtils';
+import styles from './Tabs.scss';
+
+const cx = classNames.bind(styles);
+
+/**
+NOTE: This is being commented out until descussions have been resolved around if modular tabs should be removed.
+const variants = {
+  MODULAR_CENTERED: 'modular-centered',
+  MODULAR_LEFT_ALIGNED: 'modular-left-aligned',
+  STRUCTURAL: 'structural',
+};
+*/
+
+const propTypes = {
+
+  /**
+   * Tabs style. One of: Tabs.Opts.Variants.MODULAR_CENTERED, Tabs.Opts.Variants.MODULAR_LEFT_ALIGNED, or Tabs.Opts.Variants.STRUCTURAL.
+   * NOTE: This is being commented out until descussions have been resolved around if we want modular tabs.
+  variant: PropTypes.oneOf([variants.MODULAR_CENTERED, variants.MODULAR_LEFT_ALIGNED, variants.STRUCTURAL]),
+  */
+
+  /**
+   * Indicates if tabs should fill the width available in the tab bar.
```

Comment:
If there are multiple with tabFill, will each be given equal fill widths?

Reply:
This prop is for the `Tabs` component, not the `Tabs.Pane`. When true, it applies to all the tabs. It doesn't necessarily give tabs equal widths, but the remaining space is spread evenly between the tabs (either as space-between, or stretched depending on the theme).